android/app/src/main/java/app/organicmaps/search/SearchFragment.java

```diff
@@ -398,9 +398,8 @@ void showSingleResultOnMap(@NonNull SearchResult result, int resultIndex)
     if (RoutingController.get().isWaitingPoiPick())
     {
       SearchResult.Description description = result.description;
-      String subtitle = description != null
-          ? Utils.getLocalizedFeatureType(requireContext(), description.featureType)
-          : "";
+      String subtitle = (description != null) ? description.featureType : "";
+      /// @todo Hm, is it ok here? Empty title if name is not empty?
       String title = TextUtils.isEmpty(result.name) ? subtitle : "";
```

Comment:
```suggestion
      String title = TextUtils.isEmpty(result.name) ? subtitle : result.name;
```

Should use the name when name is not empty